client/asset/eth/eth.go

```diff
@@ -1230,18 +1240,21 @@ func (eth *ExchangeWallet) SyncStatus() (bool, float32, error) {
 		}
 		return false, ratio, nil
 	}
+
+	// According to syncProgress we are at the highest network block, but check
+	// the time since the block.
 	bh, err := eth.node.bestHeader(eth.ctx)
 	if err != nil {
 		return false, 0, err
 	}
 	// Time in the header is in seconds.
-	nowInSecs := time.Now().Unix() / 1000
-	timeDiff := nowInSecs - int64(bh.Time)
-	var progress float32
-	if timeDiff < dexeth.MaxBlockInterval {
-		progress = 1
+	timeDiff := time.Now().Unix() - int64(bh.Time)
+	if timeDiff > dexeth.MaxBlockInterval && eth.net != dex.Simnet {
```

Comment:
@JoeGruffins I noticed this was dividing now in seconds by a thousand before comparing with the header time in seconds, so in practice this check was almost impossible to fail.  I'm pretty sure 180 seconds since last block in our simnet workflows is gonna be too tight so I'm excluding simnet here.  Note that the unit tests leave net as 0 (mainnet), so we're still testing the conditions.